docs/branding/logos.md

```diff
@@ -0,0 +1,31 @@
+---
+id: logos
+title: Logo
+---
+
+:::caution
+
+This section is a work in progress.
+
+:::
+
+## Idea
+Our logo is made up of 2 shapes, circle & rectangle, it represents,
+
+1. "**Talawa**" in the Jamaican dialect means "**Strong**", and there is a saying "**Little, but Talawa**", which translates to "**Little, but Strong**".
+2. We used Palisadoes Foundation logo's elements and colors to preserve the core ideologies.
+
+## Clear space
+The logo needs to have clear space, free of text, pictures, and other elements.
+
+![Clear Space](./assets/Logo-Guidelines.PNG)
```

Comment:
There is no reference to any of the existing logos. A number were created and submitted in a previous PR, which appears to have been canceled. Please add.